dev/tasks/python-wheels/travis.osx.yml

```diff
@@ -18,6 +18,35 @@ os: osx
 osx_image: xcode11.3
 language: generic
 
+addons:
+  homebrew:
+    packages:
+      - automake
```

Comment:
We can reduce the preinstalled dependencies because for some of them we bundled alternative, but I wouldn't like to do more roundtrips with travis before cutting RC1.